tests/test_benchmarks.py

```diff
@@ -0,0 +1,433 @@
+from pathlib import Path
+import importlib
+import json
+import uuid
+
+import pytest
+
+from bowtie import _benchmarks
+from bowtie._benchmarks import BenchmarkGroup
+from bowtie._cli import EX
+from bowtie._core import Dialect, Example, TestCase
+from bowtie._direct_connectable import Direct
+from tests.test_integration import bowtie
+
+validators = Direct.from_id("python-jsonschema").registry()
+
+benchmark_report_validator = validators.for_uri(
+    "tag:bowtie.report,2024:benchmark_report",
+)
+benchmark_report_validated, benchmark_report_invalidated = (
+    benchmark_report_validator.validated,
+    benchmark_report_validator.invalidated,
+)
+
+benchmark_validator = validators.for_uri(
+    "tag:bowtie.report,2024:benchmarks",
+)
+benchmark_validated, benchmark_invalidated = (
+    benchmark_validator.validated,
+    benchmark_validator.invalidated,
+)
+
+bowtie_dir = Path(__file__).parent.parent.joinpath("bowtie")
+default_benchmarks_dir = bowtie_dir.joinpath("benchmarks")
+keyword_benchmarks_dir = bowtie_dir.joinpath("benchmarks").joinpath("keywords")
+
+DIRECT_CONNECTABLE = "python-jsonschema"
+
+
+@pytest.fixture()
+def valid_single_benchmark():
+    module_name = "tests.benchmarks"
```

Comment:
There's nothing dynamic here, why avoid the `import` statement?

Reply:
Ended up following the same method here too. You are right, it isnt needed here.